index.bs

```diff
@@ -137,6 +137,79 @@ Scope {#scope}
 
 This specification is limited to providing DOM events for retrieving information describing the physical orientation and motion of the hosting device. The intended purpose of this API is to enable simple use cases such as those in [[#use-cases|Use-Cases]] section. The scope of this specification does not include providing utilities to manipulate this data, such as transformation libraries. Nor does it include providing access to low sensor data, or direct control of these sensors.
 
+Model {#model}
+=====
+
+Device Orientation {#device-orientation-model}
+------------------
+
+This specification expresses a device's physical orientation as a series of rotations relative to an <a>implementation-defined</a> reference coordinate frame.
+
+The sequence of rotation steps is a set of intrinsic Tait-Bryan angles of type Z - X' - Y'' ([[EULERANGLES]]) that are applied on the <a>device coordinate system</a> defined in [[!ACCELEROMETER]] and summarized below:
```

Comment:
I'm nervous about referencing a definition from a specification that is less far along the standards track than this one. I am curious for @anssiko's opinion here.

Reply:
I assume this is about the [device coordinate system](https://www.w3.org/TR/accelerometer/#device-coordinate-system) definition. We can use these three factors to assess the associated risk for referencing this definition: stability, schedule and licensing (from the [normative references](https://www.w3.org/Guide/process/tilt/normative-references.html) guide):
- Stability:
  - Stability of the referenced document: The WG believes the specification is stable and has demonstrated it is implementable but awaits further implementation experience before advancing to a Proposed Rec. 
  - Stability of the referenced part(s): This definition refers to an established coordinate system, a Cartesian coordinate system, and the right-hand rule, a common convention for the orientation of axes in 3D space.
  - Nature of the dependency: A coordinate system convention.
  - Status of implementations: Chromium implementations, automated tests.
- Schedule: The Accelerometer specification is a Candidate Recommendation Draft.
- Licensing: The Accelerometer specification uses a permissive document license, the same as this specification.

Considering these three factors, I think it is appropriate to incorporate this [device coordinate system](https://www.w3.org/TR/accelerometer/#device-coordinate-system) definition as a reference. If this definition would go away for some reason, it would be relatively easy to copy-paste this definition into this specification without delaying possible future transition significantly.

@himorin, let us know if you think otherwise.

Reply:
Thanks for the link to that guide. I think given the nature of the reference (definitions of sensor concepts, rather than anything specific to the Generic Sensor API) this is fine.